app/resonant-laboratory/models/Dataset.js

```diff
@@ -1,3 +1,4 @@
+import d3 from 'd3';
```

Comment:
Small style concern - the "modern" d3 way to do this is something like `import format from 'd3-format';` and then use `format()` directly in the code. What do you think?